ipay88/build.gradle

```diff
@@ -6,8 +6,9 @@ plugins {
 }
 
 android {
-    namespace 'io.primer.ipay88'
-    compileSdkVersion 34
+    // in order not to crash their SDK, we can use their namespace to correctly find R class
+    namespace 'com.androids.ipay'
```

Comment:
There are solutions for this:

- unpack their code and load resources dynamically, then package it again
- unpack their code and put it inside our package - but some things should not be pushed or added to public repo

At the end this seems as the easiest solution.